gensrc/proto/internal_service.proto

```diff
@@ -90,6 +90,16 @@ message PTabletWriterOpenResult {
     required PStatus status = 1;
 };
 
+message PartitionOpenRequest{
```

Comment:
rename it to PPartitionOpenRequest/Result as P-prefix will indicate it is a protobuf class